tests/cases/fourslash/renameDestructuringAssignmentInForOf2.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path='fourslash.ts' />
+
+////interface I {
+////    property1: number;

```

Comment:
Becauyse you are finding ref on var property1 which is different from property property1